db/migrate/20181025230931_add_comments_to_vms_table.rb

```diff
@@ -0,0 +1,102 @@
+    change_column_comment :vms, :ems_ref, "A native unique reference for the VM within the EMS."
+    change_column_comment :vms, :ems_ref_obj, "A YAML serialized version of the ems_ref."
+    change_column_comment :vms, :ems_cluster_id, "ID of associated cluster."
+
+    change_column_comment :vms, :fault_tolerance, "Indicates whether or not the VM is fault tolerant, i.e. if one host "\
+                                                  "crashes it will immediately pick up on another host in the cluster."
+
+    change_column_comment :vms, :flavor_id, "ID of associated flavor."
+    change_column_comment :vms, :format, "The format of the VM's disk, such as VMDK."
+    change_column_comment :vms, :guid, "A value used to uniquely identify the VM internally."
+    change_column_comment :vms, :id, "The internal database ID."
+    change_column_comment :vms, :host_id, "ID of the host that the VM/Template resides on."
+    change_column_comment :vms, :last_perf_capture_on, "Timestamp for last performance metrics capture."
+    change_column_comment :vms, :last_scan_on, "Timestamp for last SmartState analysis."
+    change_column_comment :vms, :last_scan_attempt_on, "Timestamp for last attempt to perform SmartState Analysis."
+    change_column_comment :vms, :last_sync_on, "Timestamp for last SmartState analysis."
+
+    change_column_comment :vms, :linked_clone, "Indicates whether the VM is a linked clone, i.e its disk is a referencing "\
+                                               "disk pointing to snapshot of another VM."
+
+    change_column_comment :vms, :location, "The location or region of the VM."
+    change_column_comment :vms, :memory_hot_add_enabled, "Indicates whether or not memory can be hot added to the running VM."
+
+    change_column_comment :vms, :memory_hot_add_increment, "Memory in MB that can be added to a running VM must be in "\
+                                                           "increments of this value. VMWare only."
+
+    change_column_comment :vms, :memory_hot_add_limit, "The maximum amount of memory, in MB, than can be added to a running VM. VMWare only."
+    change_column_comment :vms, :memory_limit, "The absolute memory limit for the VM."
+    change_column_comment :vms, :memory_reserve, "Amount of memory that is guaranteed available for the VM. VMWare only."
+
+    change_column_comment :vms, :memory_reserve_expand, "Amount the memory can grow beyond its normal limits if there "\
+                                                     "are unreserved resources available. VMWare only."
+
+    change_column_comment :vms, :memory_shares, "The number of memory shares allocated. Used to determine resource allocation "\
+                                             "in case of resource contention. VMWare only."
+
+    change_column_comment :vms, :memory_shares_level, "Memory shares allocation level. VMWare only."
+
+    change_column_comment :vms, :miq_group_id, "MIQ Group associated with the VM."
```

Comment:
Should this be similar to evm_owner_id, since this is about ownership (I think...@gtanzillo?)?  So maybe "ID of MiqGroup owner." or something?

Reply:
yes it is about group ownership. (screen "set ownership")